.github/workflows/test.yml

```diff
@@ -3,24 +3,21 @@ on: [pull_request]
 jobs:
   main:
     runs-on: ubuntu-latest
-    env:
-      CYPRESS_PLUGIN_VISUAL_REGRESSION_UPDATE_IMAGES: ${{contains(github.event.head_commit.message, ' [update screenshots]')}}
```

Comment:
I find out that there is no one-liner to get the last commit message, so I'm switching to a different strategy: commit e2e artifacts, and then the developer needs to manually pick the right screenshots and restart tests.
https://github.com/orgs/community/discussions/28474